### DISCUSSION TITLE
【求助】内容如下   委托奖励：-1600原石/星穹/菲林

### DISCUSSION THREAD

**Discussion by opqphp** [share121/inter-knot#3312] [其他]
法厄同们  为什么我每次进绳网就得清一次数据重新登录啊   有没有办法解决啊
![illust_116799150_20240819_000712](https://github.com/user-attachments/assets/142c2a5b-c869-4a5e-b312-bbc3f320022c)

**Comment by Abandon12393** [comment]
在登陆github的界面选上记住密码自动登录试试呢<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by opqphp** [reply]
#3283←像这样  重新进没有用 得清完数据再重新登录

**Reply by opqphp** [reply]
我也是移动端的

**Comment by Simlalsy** [comment]
我不会
